src/Symfony/Component/Filesystem/Filesystem.php

```diff
@@ -59,7 +59,7 @@ public function copy($originFile, $targetFile, $overwriteNewerFiles = false)
             }
 
             // Stream context created to allow files overwrite when using FTP stream wrapper - disabled by default
-            if (false === $target = @fopen($targetFile, 'w', null, stream_context_create(['ftp' => ['overwrite' => true]]))) {
+            if (false === $target = @fopen($targetFile, 'w', false, stream_context_create(['ftp' => ['overwrite' => true]]))) {
```

Comment:
Third parameter to [fopen](https://www.php.net/manual/en/function.fopen.php) bust be a boolean. Default is `false´.

From the docs: 

> The optional third use_include_path parameter can be set to '1' or true if you want to search for the file in the include_path, too.


Please very this extra carefully.